repos.json

```diff
@@ -233,6 +233,10 @@
         "vdemeester": {
             "url": "https://github.com/vdemeester/nur-packages"
         },
+        "xe": {
+            "type": "gitea",
```

Comment:
We don't support gitea yet, but maybe in future.
Right now it is just using fetchgit.

Reply:
Fixed in https://github.com/nix-community/NUR/pull/193/commits/a928253952d312052f914326d713ab5dc3d52c8d